### PR TITLE
CLN: assorted cleanups

### PR DESCRIPTION
**Pull request by jbrockmendel**

### CODE REVIEW

File: pandas/tests/dtypes/test_inference.py [hunk 1]
```diff
@@ -1123,18 +1123,15 @@ def test_is_string_array(self):
...
-            record = namedtuple("record", "x y")
-            r = record(5, 6)
-            values = [r]
-            result = lib.to_object_array_tuples(values)  # noqa
-        except ImportError:
-            pass
+        lib.to_object_array_tuples(values)
+
+        # make sure record array works
+        from collections import namedtuple
```
Comment (jreback) [inline pandas/tests/dtypes/test_inference.py]: can you move to top

File: pandas/tests/dtypes/test_inference.py [hunk 2]
```diff
@@ -1123,18 +1123,15 @@ def test_is_string_array(self):
...
+
+        record = namedtuple("record", "x y")
+        r = record(5, 6)
+        values = [r]
+        lib.to_object_array_tuples(values)
```
Comment (jreback) [inline pandas/tests/dtypes/test_inference.py]: would be nice to compare if you can

**Review by jreback — APPROVED** [review]
k, minor comment, can be done later though.

**Comment by jreback** [comment]
lgtm. ping on green.

**Comment by jbrockmendel** [comment]
ping

**Comment by jreback** [comment]
thanks

**Pull request merged**